security/middleware.py

```diff
@@ -265,7 +265,7 @@ class ClearSiteDataMiddleware(BaseMiddleware):
     REQUIRED_SETTINGS = ('CLEAR_SITE_DATA_URL_WHITELIST',)
     OPTIONAL_SETTINGS = ('CLEAR_SITE_DATA_DIRECTIVES')
 
-    DEFAULT_DIRECTIVES = ['*']
+    DEFAULT_DIRECTIVES = ['cookies', 'storage']
```

Comment:
We need to update our side to not have any `CLEAR_SITE_DATA_DIRECTIVES` otherwise it'll still use the old values:
`CLEAR_SITE_DATA_DIRECTIVES = ['cache', 'cookies', 'executionContexts']` I think your change will only change if the CLEAR_SITE_DATA_DIRECTIVES value is None.